pyriemann/utils/rotate.py

```diff
@@ -0,0 +1,233 @@
+import warnings
+import numpy as np
+from ..utils.distance import distance
```

Comment:
I feel this utils submodule is getting too crowded. I would create an pyriemann/transfer folder and put this file in pyriemann/transfer/_rotate.py
(starting with _ if it's meant to be private functions only).